### DISCUSSION TITLE
Help with using a menu

### DISCUSSION THREAD

**Discussion by ashleygrobler04** [Ghorthalon/audiogame-tools#15] [Q&A] [answered]
Hi all. I transpiled the engine to js and am now using it in a vanilla js project. I am trying to make a menu to test out the features.
I am running to an issue when loading sounds for the menu. I get the error saying play is not a function. 
I will be posting my code. Is the way I am initializing the menu wrong?
If so, what is the right way that I should initialize the menu? Especially the soundset bit. 
import { Menu } from "./dist/ui/menu/index.js";
import { MenuItem } from "./dist/ui/menu/index.js";

const m = new Menu(
  "Test menu",
  [
    new MenuItem("1", "Option 1"),
    new MenuItem("2", "Option 2"),
    new MenuItem("3", "Option 3"),
  ],
  {
    move: "./s.mp3",
    boundary: "./s.mp3",
    open: "./s.mp3",
    close: "./s.mp3",
    choose: "./s.mp3",
  }
);
m.run(document.getElementById("menu"));
Hope to receive a response...

**Comment by Ghorthalon** [comment, marked as answer]
The sounds are expected to be of any sound type with a .play() method. This could be a media element, or something from resonator, howler, etc.
So instead of just passing the file names, you could do something like
move: new Audio("./s.mp3"),
boundary: new Audio("./s.mp3"),
open: new Audio("./s.mp3")
And so on.
That should work.
It's not ideal. I'm honestly not too happy with the entire menu system in general.

**Comment by ashleygrobler04** [comment]
Got it, thanks.
